venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,780 @@
+                ],
+                writers = [
+                    'aclweb.org/ACL/2022/Conference'
+                ],
+                content = arr_review.content
+            )
+            acl_review_posted = client.post_note(acl_review)
+            assert acl_review_posted, print('failed to post review ', acl_review.id)
+            acl_reviews_dictionary[acl_review_posted.signatures[0]] = acl_review_posted.replyto
+            post_metareviews(acl_blind_submission_forum, acl_blind_submission, arr_submission)
+        
+# Post metareviews 
+def post_metareviews(acl_blind_submission_forum, acl_blind_submission, arr_submission):
+    # Migrate all reviews from the original ARR Submission 
+    # Get invitation for that month's reviews 
+    conf_id = arr_submission.invitation.rsplit('/', 2)[0]
+    metareview_invitation_arr = '{conf_id}/Paper{number}/-/Meta_Review'.format(conf_id = conf_id,number = arr_submission.number)
+    
+    # Get all reviews from the original ARR Submission 
+    arr_metareviews = openreview.tools.iterget_notes(client, invitation = metareview_invitation_arr)
+    
+    # Iterate through each review and for each, create and post a new review 
+    for arr_metareview in arr_metareviews:
+        if(arr_metareview.signatures[0] not in acl_metareviews_dictionary):
+            acl_metareview = openreview.Note(
+                forum = acl_blind_submission_forum,
+                replyto = acl_blind_submission_forum,
+                invitation = f'aclweb.org/ACL/2022/Conference/-/Meta_Review',
+                signatures = arr_metareview.signatures,
+                readers = [
+                    'aclweb.org/ACL/2022/Conference',
+                    'aclweb.org/ACL/2022/Conference/Program_Chairs'
+                ],
+                writers = [
+                    'aclweb.org/ACL/2022/Conference'
+                ],
+                content = arr_metareview.content
+            )
+            acl_metareview_posted = client.post_note(acl_metareview)
+            assert acl_metareview_posted, print('failed to post metareview ', acl_metareview.id)
+            acl_metareviews_dictionary[acl_metareview_posted.signatures[0]] = acl_metareview_posted.replyto
+            
+# Retrieve all commitment submissions, ACL submissions, ACL blind submissions, and ACL blind submission reviews  
+commitment_notes = list(openreview.tools.iterget_notes(client,invitation='aclweb.org/ACL/2022/Conference/-/Commitment_Submission'))
+acl_submissions = list(openreview.tools.iterget_notes(client,invitation='aclweb.org/ACL/2022/Conference/-/Submission'))
+blind_submissions = {note.original: note for note in list(openreview.tools.iterget_notes(client, invitation = 'aclweb.org/ACL/2022/Conference/-/Blind_Submission'))}
+acl_reviews_dictionary = {review.signature : review.replyto for reviews in list(openreview.tools.iterget_notes(client, invitation = 'aclweb.org/ACL/2022/Conference/Paper.*/-/Official_Review'))}
```

Comment:
I think we are using `aclweb.org/ACL/2022/Conference/-/Official_Review` as the review invitation for ACL submissions, so we don't need to add the paper number to get all reviews